spec/features/applications/show_spec.rb

```diff
@@ -60,4 +60,64 @@
     expect(page).to_not have_content("Adoptable")
     expect(page).to have_content("On hold for Joe Smith")
   end
+
+  it "I cannot see approve application if any of the pet's status equals Adoptable." do
```

Comment:
Should this be cannot see approve application if any of the pet's status equals Pending Adoption? If they are Adoptable we should be able to see the link.